packages/core/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "devDependencies": {
     "@rslib/tsconfig": "workspace:*",
+    "@rspack/core": "1.0.8",
```

Comment:
We could use `import type { Rspack } from '@rsbuild/core';` to get the Rspack type from Rsbuild.

Reply:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/912a344c-f842-4851-9421-2326a3e4a2d7">

It's still difficult to get this type.